lib/api/src/grpc/proto/collections.proto

```diff
@@ -316,6 +317,7 @@ message UpdateCollection {
   optional VectorsConfigDiff vectors_config = 6; // New vector parameters
   optional QuantizationConfigDiff quantization_config = 7; // Quantization configuration of vector
   optional SparseVectorConfig sparse_vectors_config = 8; // New sparse vector parameters
+  map<string, string> properties = 9; // New properties of the collection
```

Comment:
I would prefer to have a single optional string called `comment`

Reply:
resolved.